cfme/tests/webui/test_advanced_search.py

```diff
@@ -176,18 +176,13 @@ def test_can_open_advanced_search(param, appliance):
     view.search.close_advanced_search()
 
 
-@pytest.mark.uncollectif(lambda param, appliance: (
-    (param.collection in (MyService, 'physical_providers', 'physical_servers', 'volume_backups',
-                          'volume_snapshots') and appliance.version < '5.9') or
-    (param.collection in (ConfigManager, 'ansible_tower_providers') and appliance.version > '5.10')
-    or (param.filter == 'Template (Ansible Tower) : Name' and appliance.version < '5.10') or
-    (param.filter == 'Job Template (Ansible Tower) : Name' and appliance.version > '5.10') or
-    (param.collection in ('cloud_host_aggregates', 'cloud_instances', 'cloud_images', 'infra_vms',
-                          'infra_templates', 'datastores', ConfigManager, 'ansible_tower_providers',
-                          'ansible_tower_systems', 'ansible_tower_job_templates', VmsInstances)
-    and appliance.version < '5.10'))
-    or (param.collection == TemplatesImages and BZ(1626579)) or (param.collection == MyService
-                                                                 and BZ(1627078)))
+@pytest.mark.uncollectif(
+    lambda param, appliance: (
+        appliance.version > '5.10' and
```

Comment:
looks like `appliance.version` is not needed here

Reply:
I think it is, since we're testing 5.10+ right now. Per the original uncollect, this is looking for `>`, not `>=`. I'm not 100% on specifics here, this may be something that is in a 5.10 z-stream and this should be more explicitly checking `>= 5.10.2`

Reply:
@lina-nikiforova Can you double check - do you agree or am I missing something here? Thanks!

Reply:
@mshriver I think the original uncollect is not accurate as it doesn't catch `5.10`, only `>` or `<` are used.

Also, any 5.10.x > 5.10
![image](https://user-images.githubusercontent.com/42433123/55785534-e38f2b00-5ab2-11e9-957c-bb939d900a3c.png)